INCHI-1-SRC/INCHI_BASE/src/inchi_api.h

```diff
@@ -101,8 +101,8 @@ typedef enum tagINCHIBondStereo2D {
  * whether the stereochemistry is unknown.                               *
  *************************************************************************/
 
-/* sizes definitions */
-#define MAXVAL                   20 /* max number of bonds per atom                 */
+ /* sizes definitions */
+#define MAXVAL                   50 /* max number of bonds per atom                 */ /* @nnuk : Changed from 20 -> 50 for testing purposes, to account for some structures with one atom having more than 20 bonds */
```

Comment:
A search of the codebase shows > 80 occurrences of `MAXVAL`, being used in guards, for array sizing and comments. How do we ensure that changing `MAXVAL` doesn't result in unintended consequences?
For examples, some comments are now incorrect such as
https://github.com/IUPAC-InChI/InChI/blob/e0b35b0e61814299c631da5dbad9fbaae0e0b930/INCHI-1-SRC/INCHI_BASE/src/ichiprt1.c#L2921
https://github.com/IUPAC-InChI/InChI/blob/e0b35b0e61814299c631da5dbad9fbaae0e0b930/INCHI-1-SRC/INCHI_BASE/src/mol_fmt1.c#L394
https://github.com/IUPAC-InChI/InChI/blob/e0b35b0e61814299c631da5dbad9fbaae0e0b930/INCHI-1-SRC/INCHI_BASE/src/mol2atom.c#L438